package/myrlyn.spec

```diff
@@ -36,7 +36,7 @@ BuildRequires:  hicolor-icon-theme
 # FIXME: Is this still needed (gcc-fortran)?
 BuildRequires:  gcc-fortran
 BuildRequires:  pkgconfig
-BuildRequires:  pkgconfig(Qt6Core)
+BuildRequires:  pkgconfig(Qt6Core) >= 6.5
```

Comment:
I admit I was sceptical about this, but I tried in my OBS home project _myrlyn-git_ subproject, and it actually worked.